.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,770 @@
+
+   **Tip:** If you get `make: *** No rule to make target 'html'.  Stop.` error message after running the `make html` command, you should check if you're in the correct directory. You must be in the `docs/` directory to run this command.
+3. Click the preview button — resembles a book and a magnifying glass — at the top to trigger Esbonio, a tool used for live preview. A tab opens, but the preview won't work. You can safely close this tab.
+
+   ![Highlight preview button on the top bar of VS Code on codespace](../assets/images/preview_button_vscode_codespace.png)
+
+4. At the bottom panel, click the 'Ports' tab.
+5. Click the globe icon to open the live preview in your browser. Now you can see the project in real-time on localhost.
+
+   ![Highlight port tab and globe icon to open preview in browser at VS Code on codespace](../assets/images/port_and_open_browser_vscode_codespace.png)
+
+<br />
+
+<details>
+  <summary id="troubleshooting-live-preview"><strong>Troubleshooting live preview</strong></summary>
+
+  <br />
+  <p><strong>Troubleshooting #1</strong></p>
+  <p>If you can't see your changes in the live preview, try refreshing the page.</p>
+
+  <p><strong>Troubleshooting #2</strong></p>
+  <p>If refreshing doesn't work, try to:</p>
+  <ol>
+    <li>Delete the <code>build</code> folder in the root.</li>
+    <li>Delete the <code>build</code> folder in the <code>docs</code> directory.</li>
+    <li>Refresh your codespace browser.</li>
+    <li>Ensure that you're in the <code>docs/</code> directory.</li>
+    <li>Follow the steps in the <a href="#live-preview-on-codespace">Live Preview on Codespace</a> section.</li>
+  </ol>
+
+  <p><strong>Troubleshooting #3</strong></p>
+  <p>If the above steps fail:</p>
+  <ol>
+    <li>Close VS Code and the live preview browsers.</li>
+    <li>Go to <a href="https://github.com/codespaces" target="_blank">https://github.com/codespaces</a>.</li>
+    <li>At the bottom, you should see a list of your codespaces.</li>
+    <li>Click the three dots icon on the right of your project's codespace.</li>
+    <li>Click 'Stop codespace'.</li>
+    <li>Re-open the codespace by clicking its name.</li>
+    <li>Follow the steps in the <a href="#live-preview-on-codespace">Live Preview on Codespace</a> section.</li>
+  </ol>
+</details>
+
+<br />
+
+> [!TIP]
+>
+> - Always refresh the page to view the new changes you have applied.
+> - All commands only work within the `docs/`directory. If you're unable to run a command, verify that you're in the correct directory.
+> - Read the "[Troubleshooting live preview](#troubleshooting-live-preview)" section if you encounter any issues with the live preview in the codespace.
+
+<br />
+
+If you're ready to push your changes to the remote repository and create a PR, please read the "[Push changes to the remote repository](#push-changes-to-the-remote-repository)" and "[Create a PR](#create-a-pr)" sections.
+
+### 3. Local development
```

Comment:
I feel like for the dev docs we should lead with working locally, perhaps? Up to you though! :)